src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h

```diff
@@ -12,12 +12,17 @@
 #define GPIO_PIN_RST            14
 #define GPIO_PIN_RX_ENABLE      27
 #define GPIO_PIN_TX_ENABLE      26
-#define GPIO_PIN_OLED_SDA       -1
-#define GPIO_PIN_OLED_SCK       -1
+#define GPIO_PIN_OLED_CS 15
+#define GPIO_PIN_OLED_RST 16
+#define GPIO_PIN_OLED_DC 17
+#define GPIO_PIN_OLED_MOSI 32
+#define GPIO_PIN_OLED_SCK 33
 #define GPIO_PIN_RCSIGNAL_RX    13
 #define GPIO_PIN_RCSIGNAL_TX    13
 #define GPIO_PIN_LED_WS2812     15
 #define GPIO_PIN_FAN_EN         17
+#define HAS_OLED
+#define HAS_OLED_SPI
```

Comment:
Should HAS_OLED and HAS_OLED_SPI be in the target .ini?

Reply:
So, I think these should be in the targets.. they are hardware specific and I think we also have some packback stuff in a few of those other target.h files. 

I moved them up to a nice `// Any device features` section.